pkgs/applications/radio/tncattach/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchFromGitHub, installShellFiles }:
+
+stdenv.mkDerivation rec {
+  pname = "tncattach";
+  version = "0.1.9";
+
+  src = fetchFromGitHub {
+    owner = "markqvist";
+    repo = "tncattach";
+    rev = "bb9ff10158c04a37cb3221152885a083e3059909";
+    sha256 = "0n7ad4gqvpgabw2i67s51lfz386wmv0cvnhxq9ygxpsqmx9aynxk";
+  };
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  installPhase = ''
+    install -Dm755 tncattach -t $out/bin
+    installManPage tncattach.8
+  '';
+
+  meta = with lib; {
+    description = "Attach KISS TNC devices as network interfaces";
+    homepage = "https://github.com/markqvist/tncattach";
+    license = licenses.mit;
+    maintainers = with maintainers; [ sarcasticadmin ];
+    platforms = platforms.unix;
```

```suggestion
    platforms = platforms.linux;
```
From README:
> Attach KISS TNC devices as network interfaces in Linux.

So, I guess it's only for linux. Correct me if I'm wrong.

---

Im unsure, but I dont have anything at the moment to validate that it works outside of linux. Will change this to `linux` for now

---

I've just tried to build on Darwin:
```
tncattach> Compiling with: clang
tncattach> clang -Wall -std=gnu11 -static-libgcc tncattach.c Serial.c TCP.c KISS.c TAP.c -o tncattach -Wall
tncattach> clang-11: error: unsupported option '-static-libgcc'
tncattach> make: *** [makefile:17: tncattach] Error 1
```